src/ccm/ng-ce-ui/templates/deployment.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ng-ce-ui
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "ng-ce-ui.labels" . | nindent 4 }}
+spec:
+  {{- if not .Values.autoscaling.enabled }}
+  replicas: {{ .Values.replicaCount }}
+  {{- end }}
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: {{.Values.maxSurge}}
+      maxUnavailable: {{.Values.maxUnavailable}}
+  selector:
+    matchLabels:
+      {{- include "ng-ce-ui.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        app: ng-ce-ui
+        {{- include "ng-ce-ui.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
```

Comment:
We can add a function for image Pull secrets: like https://github.com/harness/helm-platform/blob/main/src/harness-manager/templates/deployment.yaml#L32 and helper function to be added in helper.tpl

Reply:
Added this, kindly check!